bin/pycbc_single_template

```diff
@@ -415,9 +415,9 @@ with ctx:
     f['chisq'].attrs['delta_t'] = snr.delta_t
     if opt.trigger_time is not None:
         f.attrs['event_time'] = opt.trigger_time
-    f.attrs['approximant'] = approximant
-    f.attrs['ifo'] = ifo
-    f.attrs['command_line'] = ' '.join(sys.argv)
+    f.attrs['approximant'] = approximant.encode()
```

Comment:
Ok, this one is going to trip me up a lot...

Reply:
Yeah, took a little while to figure out the problem here! You can use `str` values as `keys` in a HDF file, but not as a `value`. This always needs translating to `bytes`.

Reply:
I'm taking the approach of encoding on writing, and decoding on reading, and then assuming you are dealing with strings apart from that.